libs/libcommon/src/libcommon/simple_cache.py

```diff
@@ -79,6 +80,7 @@ class CachedResponse(Document):
         updated_at (`datetime`): When the cache entry has been last updated.
         worker_version (`str`): The semver version of the worker that cached the response.
         dataset_git_revision (`str`): The commit (of the git dataset repo) used to generate the response.
+        progress (`float`): Progress percentage if the result is not complete yet.
```

Comment:
nit: percentage seems to imply that the value goes between 0 and 100. But the alternative names are worse (https://english.stackexchange.com/questions/275734/a-word-for-a-value-between-0-and-1-inclusive) so I think it's better to keep percentage 😛 (the type makes it clear below)